specification/specification.md

```diff
@@ -217,6 +217,42 @@ range for each execution domain. Software configurations are explained
 below.
 
 
+Secure Addresses
+================
+
+It is possible for a single device to be accessible at different
+addresses whether the transaction is marked as secure or non-secure.
+
+The normal world address range of the device is specified used "reg" as
+usual. If the device comes with a different address range for secure
+world accesses then it can be specified using "secure-reg":
```

Comment:
I thought we had abandoned this idea?

Reply:
Thanks for taking a look! You are right, I missed an update. I'll do that and send a new pull request also including the other changes you suggested.